heat/core/statistics.py

```diff
@@ -569,6 +704,15 @@ def mpi_argmin(a, b, _):
 
 MPI_ARGMIN = MPI.Op.Create(mpi_argmin, commute=True)
 
+# # TODO: implement mpi_minimum
```

Comment:
Does this need to be here? this could be put in an issue

Reply:
Sorry about this, it's a leftover from the attempted implementation through __reduce_op(), I removed it already.